### ISSUE TITLE
v1.07 in excel 2010, Offset.  (also tooltop displaying IntelliSence by...)

### ISSUE THREAD

**Issue by nVenticDW** [Excel-DNA/IntelliSense#61]
Hi

firstly I want to say that this is an excellent dll.
 
v1.07 in Excel 2010 (32 bit) appears to "offset". That means to say  1.07 needs the the cursor right of the actual text of the function name (on top of the first parameter).  The same in Excel 2016 as well

In 1.06: when the intellisense tool comes up when entering the function followed by the open bracket, the function name itself is underlined and when ones move the mouse to it, a hand is shown and when one clicks the help file entry is displayed. All perfect for the function name. 

In v1.07 in Excel 2010: when the intellisense tool comes up when entering the function followed by the open bracket, the function name itself is underlined and when ones move the mouse to it the underline goes and a four ended arrow is shown. One must click to the right (offset)  on top of the first parameter for the help file entry to be displayed.  is this by design? 

I have taken photos of the screen  (screenshots don't pick up the cursor). Hopefully this makes the issue clear (assuming it is not by design).

![v106 img_2815](https://user-images.githubusercontent.com/20855083/39470338-6a410206-4d3d-11e8-8742-206cb7231b0b.jpg)
![v107 offset img_2813](https://user-images.githubusercontent.com/20855083/39470339-6a60c082-4d3d-11e8-9216-dd6e677d463a.jpg)
![v107 fourarrow img_2814](https://user-images.githubusercontent.com/20855083/39470340-6a7a72b6-4d3d-11e8-9500-b2a7cce27615.jpg)

I have also noticed that the tooltip always displays  "Intellisence by Excel-DNA" now when one hovers over a parameter (in both 1.06 and 1.07), I seem to remember it used to show the one line description of the parameter. There has been so many developments in the past days. Is this something that will always be there or is there perhaps a change in usage and I have to change the _IntelliSense_ worksheet. 
I see in B1, I now have to enter the number 1 but that hasn't made a difference (I also formatted it as a number).

Dan

**Comment by govert** [comment]
@nVenticDW I see the offset bug that you report - I think I broke it when I changed the layout code for version 1.0.7. The intended behaviour is that the blue color and underline is shown on the function name when the mouse hovers over the name, just like the built-in functions in Excel, as it was in v 1.0.6. 

The "IntelliSense by Excel-DNA" tooltip has always been there as a little advertisement - I don't think anything changed in that regard.
In future we might try to emulate Excel, which makes each parameter in the IntelliSense display into a hyperlink, and if you click on it selects that part of the formula. But that's not implemented at the moment.
